src/components/views/EResource.js

```diff
@@ -1,32 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
 
-import { LoadingPane, Pane } from '@folio/stripes/components';
-import { TitleManager } from '@folio/stripes/core';
+import { Button, LoadingPane, Pane, PaneMenu } from '@folio/stripes/components';
+import { IfPermission, TitleManager } from '@folio/stripes/core';
 
 import Package from './Package';
 import Title from './Title';
+import PCI from './PCI';
+import { resourceClasses } from '../../constants';
 
 export default class EResource extends React.Component {
   static propTypes = {
     data: PropTypes.shape({
       eresource: PropTypes.shape({
+        class: PropTypes.string,
         name: PropTypes.string,
         type: PropTypes.object,
       }),
     }),
     handlers: PropTypes.shape({
       onClose: PropTypes.func.isRequired,
+      onEdit: PropTypes.func.isRequired,
       onNeedMorePackageContents: PropTypes.func.isRequired,
       onToggleTags: PropTypes.func.isRequired
     }).isRequired,
     helperApp: PropTypes.node,
     isLoading: PropTypes.bool,
   }
 
+  renderEditPCIPaneMenu = () => {
+    const {
+      data: { eresource },
+      handlers,
+    } = this.props;
+
+    return (eresource?.class === resourceClasses.PCI) ? (
+      <IfPermission perm="ui-agreements.resources.edit">
+        <PaneMenu>
+          <FormattedMessage id="ui-agreements.eresources.edit">
+            {ariaLabel => (
+              <Button
+                aria-label={ariaLabel}
```

Comment:
Don't set aria-label if it's just the same thing as the button label itself. It makes the screen reader read out the same thing twice.

Reply:
Sorry, I meant to just remove aria-label. It's obvious what the Edit button is going to do, so an aria attribute isn't necessary for clarification.